### PR TITLE
Definitions updates for DSE 6.0.12, 6.7.8

### PR DESCRIPTION
**Pull request by cpmcdaniel**

### CODE REVIEW

**Comment by cpmcdaniel** [comment]
There is no reason to merge this PR until DSE 6.0.12 and 6.7.8 are released. This represents the changes in the rc1 tags.

File: resources/dse-yaml/dse/dse-yaml-dse-transforms.edn [hunk 1]
```diff
@@ -31,6 +31,18 @@
                                  :description "Directory to store insights logs. Example value: \"/var/log/cassandra/\""}}
                       :order [:data_dir :log_dir]}
                      :group "Insightful Monitoring"))
+ "6.0.12" (transforms
+           (add-field :role_management_options.stats
+                      {:type "boolean"
+                       :required false})
```
Comment (respringer) [inline resources/dse-yaml/dse/dse-yaml-dse-transforms.edn:37]: I guess the master branch of cass-config-builder expects ":required" fields, right?

Comment (cpmcdaniel) [reply resources/dse-yaml/dse/dse-yaml-dse-transforms.edn:37]: oops, it does not. I need to update this PR.

**Comment by cpmcdaniel** [comment]
I accidentally pushed my changes for DSE 6.8.1 to this same branch (master on my fork). I am thus closing this and opening a new PR with a feature branch for just 6.0.12 and 6.7.8. Since I have addressed the only review comment, will you approve it and merge?

**Pull request closed without merging**